client/app/components/routes/access-requests/AccessRequests.tsx

```diff
@@ -0,0 +1,78 @@
+import { GridRowsProp } from "@mui/x-data-grid";
+
+import { actionHandler } from "@/app/utils/actions";
+import DataGrid from "@/app/components/datagrid/DataGrid";
+import { UserOperator } from "./types";
+
+// 🛠️ Function to fetch user-operators
+async function getUserOperators() {
+  try {
+    return await actionHandler(
+      "registration/user-operators",
+      "GET",
+      "/dashboard/operators",
+    );
+  } catch (error) {
+    // Handle the error here or rethrow it to handle it at a higher level
+    throw error;
+  }
+}
+
+function capitalizeString(value: string): string {
+  return value.slice(0, 1).toUpperCase() + value.slice(1);
+}
+
+// 🧩 Main component
+export default async function AccessRequests() {
+  // Fetch userOperator data
+  const userOperators: [UserOperator] = await getUserOperators();
+  if (!userOperators) {
+    return (
+      <div>
+        No user-operators data in database (did you forget to run `make
+        loadfixtures`?)
+      </div>
```

Comment:
It's possible there actually wouldn't be user operators, so this should probably be a more user-friendly message. We can check with the designers for a proper one, but for now maybe just something like, "no access requests yet" (or whatever makes sense to you)

Reply:
changed